### PR TITLE
feat(core): add present to edgeless sharing

### PR DESCRIPTION
**Pull request by JimmFly**

### CODE REVIEW

**Comment by JimmFly** [comment]
Current dependencies on/for this PR:
* canary
  * **PR #5140** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/5140?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/toeverything/AFFiNE/5140?utm_source=stack-comment).

File: packages/frontend/core/src/components/cloud/share-header-right-item/present.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import { useCallback, useEffect, useState } from 'react';
+
+import * as styles from './styles.css';
+
+export const PresentButton = () => {
+  const t = useAFFiNEI18N();
+  const [isPresent, setIsPresent] = useState(false);
+
+  const handlePresent = useCallback(() => {
+    const editorRoot = document.querySelector('block-suite-root');
```
Comment (joooye34) [inline packages/frontend/core/src/components/cloud/share-header-right-item/present.tsx:15]: do we have api from blocksute model?

Comment (JimmFly) [reply packages/frontend/core/src/components/cloud/share-header-right-item/present.tsx:15]: I’m under the impression that we don’t have any API related to this, if my memory serves me correctly.

Comment (joooye34) [reply packages/frontend/core/src/components/cloud/share-header-right-item/present.tsx:15]: add todo comments to me

Comment (JimmFly) [reply packages/frontend/core/src/components/cloud/share-header-right-item/present.tsx:15]: In the future, an editor's Atom will be encapsulated, so let's deal with it now.

**Review by joooye34 — APPROVED** [review]

**Pull request merged**